generators/generator-base-private.js

```diff
@@ -1384,10 +1384,26 @@ module.exports = class extends Generator {
      * @param {string} clientRootFolder
      */
     getEntityParentPathAddition(clientRootFolder) {
-        if (clientRootFolder) {
-            return '../';
+        if (!clientRootFolder) {
+            return '';
+        }
+        const relative = path.relative(`/app/entities/${clientRootFolder}/`, '/app/entities/');
+        if (relative.includes('app')) {
+            // Relative path outside angular base dir.
+            const msg = `
+                "clientRootFolder outside app base dir '${clientRootFolder}'"
+            `;
+            // Test case doesn't have a enviroment instance so return 'error'
+            if (this.env === undefined) {
+                throw new Error(msg);
+            }
+            this.error(msg);
+        }
+        const addition = relative.replace(/[/]?..\/entities/, '').replace('entities', '..');
```

Comment:
I don't understand what this variable is supposed to represent, can you rename it please?

Reply:
Don't know how to explain but, is kind of relative path used by tests to look for the entities files.

Reply:
Okay, then let's rename it for what it is: `entityFolderFolderPath`